examples/CMakeLists.txt

```diff
@@ -62,6 +67,7 @@ function(assign_ir)
             -i ${INPUTS_DIR}/${input}
             ${minus_p} ${private_input_string}
             -c circuit_${target}.crct
+            -j table_pieces_${target}.json
```

Comment:
why need `table_pieces` for generate both?

Reply:
it includes generate_crct, and generate_crct generates table_pieces

Reply:
I don't like that idea, generate_both includes circuit and table generation, first one generates table_pieces. One would expect that circuit generation generates table pieces regardless of whether circuit and table are generated simultaneously or separately. contradicts the principle of least astonishment. And maybe there shouldn't be such thing as table_pieces_name at all. Just use table_pieces_circuit_file_name.crct. What do you think?